### PR TITLE
feat: initial assets uploader queue

### PR DESCRIPTION
**Pull request by toondaey**

### CODE REVIEW

File: cognite/extractorutils/uploader/assets.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+        self.latency = ASSETS_UPLOADER_LATENCY
+        self.latency_zero_point: Arrow = arrow.utcnow()
+
+    def add_to_upload_queue(self, asset: Asset) -> None:
+        """
+        Add asset to upload queue. The queue will be uploaded if the queue size is larger than the threshold
+        specified in the __init__.
+
+        Args:
+            asset: Asse to add
```
Comment (ozangoktan) [inline cognite/extractorutils/uploader/assets.py:88]:
```suggestion
            asset: Asset to add
```

File: cognite/extractorutils/uploader/assets.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+
+class AssetUploadQueue(AbstractUploadQueue):
+    """
+    Upload queue for assets
+
+    Args:
+        cdf_client: Cognite Data Fusion client to use
+        post_upload_function: A function that will be called after each upload. The function will be given one argument:
+            A list of the events that were uploaded.
```
Comment (ozangoktan) [inline cognite/extractorutils/uploader/assets.py:47]:
```suggestion
            A list of the assets that were uploaded.
```

**Review by ozangoktan — COMMENTED** [review]
Looks good, just a few typos we should fix

File: cognite/extractorutils/uploader/assets.py (hunk 2 above)
Comment (toondaey) [reply cognite/extractorutils/uploader/assets.py:47]: You got me, shamelessly borrowing from events. 🙈

**Review by ozangoktan — APPROVED** [review]
🚀

**Pull request merged**